lib/joke.rb

```diff
@@ -1,2 +1,10 @@
 class Joke
+
+attr_reader :id, :setup, :punchline
+
+  def initialize(id, setup, punchline)
+    @id = id
+    @setup = setup
+    @punchline = punchline
+  end
```

Comment:
excellent work in here; be sure to indent the attr_reader line, and you can also remove the blank link above it